docs/reference/plugins.rst

```diff
@@ -276,6 +276,28 @@ The default implementation is as follows::
 
 See also: `text_enabled`_, `icon_src`_
 
+text_editor_button_icon
+-----------------------
+
+When `text_enabled`_ is ``True``, this plugin can be added in a text editor and
+there might be an icon button for that purpose. This method allows to override
```

Comment:
I'd change this to. "there might be an icon button for that purpose if supported by the text plugin."